projects/commudle-admin/src/app/feature-modules/community-builds/components/community-build-h-list-item/community-build-h-list-item.component.scss

```diff
@@ -22,10 +27,18 @@
 
         p.show-details{
           margin: 0;
+          color: black;
```

Comment:
What's the natural color here?
Any color needs to be fetched from variables file